pages/osx/autopkg.md

```diff
@@ -0,0 +1,35 @@
+# autopkg
+
+> Automation framework for macOS software packaging and distribution.
+
+- List available recipe identifiers:
+
+`autopkg list-recipes`
+
+- Run autopkg recipe:
+
+`autopkg run {{recipe_identifier}}`
+
+- Run autopkg with specified pkg:
+
+`autopkg run {{recipe_identifier}} --pkg {{path/to/downloaded_pkg_or_dmg}}`
+
+- Add a new repo to autopkg:
```

Comment:
If I can add a new repo, do I need to install a recipe before using it? I'm not a macOS user, so I can't tell :P

Other than that question, looks ok to me! Thanks for the page, @Nerthil - and for being so patient with the review process 😺

Reply:
no, you don't have to use any repos for run a recipe but it's useful. 😄